lib/galaxy/managers/genomes.py

```diff
@@ -20,6 +20,13 @@ def __init__(self, app: StructuredApp):
     def get_dbkeys(self, user: m.User, chrom_info: bool) -> List[List[str]]:
         return self.genomes.get_dbkeys(user, chrom_info)
 
+    def is_registered_dbkey(self, dbkey: str, user: m.User) -> bool:
+        dbkeys = self.get_dbkeys(user, chrom_info=False)
+        for item in dbkeys:
```

Comment:
```suggestion
        for _, item in dbkeys:
```

Might also work when its only two elements.

Reply:
Nice, Danke!